drivers/led/CMakeLists.txt

```diff
@@ -4,5 +4,6 @@ zephyr_sources_ifdef(CONFIG_HT16K33 ht16k33.c)
 zephyr_sources_ifdef(CONFIG_LP3943 lp3943.c)
 zephyr_sources_ifdef(CONFIG_LP5562 lp5562.c)
 zephyr_sources_ifdef(CONFIG_PCA9633 pca9633.c)
+zephyr_sources_ifdef(CONFIG_LED_GPIO gpio.c)
```

Comment:
I'd name this file as `led_gpio` to match the config.

Reply:
Ok sounds good. Changed